scripts/test-atypical-padding.py

```diff
@@ -219,6 +233,8 @@ def main():
     node = node.add_child(ClientHelloGenerator(ciphers, extensions=ext))
     extensions = {ExtensionType.encrypt_then_mac:None,
                   ExtensionType.renegotiation_info:None}
+    if ems:
+        extensions[ExtensionType.extended_master_secret] = None
     node = node.add_child(ExpectServerHello(extensions=extensions))
```

Comment:
This is probably just my limited TLS knowledge, but.. Why are we adding EMS extension to ServerHello in couple of places, but in most cases, we are not adding it?

Reply:
if we specify the server extension in ExpectServerHello,we need to specify all the ones that server needs to send, if we don't specify any, then server can respond with any valid